pages/legal/disclaimers/disclaimers.md

```diff
@@ -1,8 +1,4 @@
 **Effective Date: July 26, 2018**
 
 The Blockstack Tokens are a crypto asset that is currently being developed by
```

Comment:
@krtb Thanks for the PR a couple of comments.  I think you went a bit too far.  The final should read:

```
The Blockstack Tokens are a crypto asset that is currently being developed by
Blockstack Token LLC, a Delaware limited liability company, whose website can be
found at [www.blockstack.com](https://www.blockstack.com). The website you are
currently visiting [(www.blockstack.org)](www.blockstack.org) is sponsored by
Blockstack PBC, an affiliate of Blockstack Token LLC, and should not be viewed
as an offer or sale of securities.
```

This is the only section `, whose website can be found at [www.blockstack.com](https://www.blockstack.com)` that needs to be removed.

BTW, you can test your changes locally before sending your PR. The Contributing section in the README explains how to test using either `npm` or `yarn`.